server/src/test/java/org/elasticsearch/common/xcontent/BaseXContentTestCase.java

```diff
@@ -525,11 +525,11 @@ public void testJavaTime() throws Exception {
         assertResult("{'date':null}", () -> builder().startObject().timeField("date", (LocalDateTime) null).endObject());
         assertResult("{'date':null}", () -> builder().startObject().field("date").timeValue((LocalDateTime) null).endObject());
         assertResult("{'date':null}", () -> builder().startObject().field("date", (LocalDateTime) null).endObject());
-        assertResult("{'d1':'2016-01-01T00:00:00.000'}",
+        assertResult("{'d1':'2016-01-01T00:00:00.000Z'}",
```

Comment:
Is this backwards compatible to what Joda did or changing some behaviour?

Reply:
this is new (LocalDateTime) in java-time. And the change is because we made it `JavaDateFormatter.from` to wrap `TemporalAccessor` into `ZonedDateTime`